lang-python/fastbencode/autobuild/defines

```diff
@@ -0,0 +1,15 @@
+PKGNAME=fastbencode
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="python-build python-installer wheel setuptools-python3 setuptools-rust llvm cython"
+PKGDES="Fast implementation of bencode"
```

Comment:
Implementation of Bencode, and encoding format used for peer-to-peer file transmission